src/index.css

```diff
@@ -1,3 +1,8 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@layer utilities {
+  .rectangle {
+    clip-path: polygon(0 0, 100% 0, 82% 100%, 0% 100%);
+  }
+}
```

Comment:
Does this makes the polygon we see on the "Dreams Will Come True" part ?

Reply:
Yes. This is for shaping the left div.